api/app/serializers/user_partial_serializer.rb

```diff
@@ -0,0 +1,8 @@
+# Serializes a Text model
+class UserPartialSerializer < ActiveModel::Serializer
```

Comment:
Same comment applies here — I don't want to go down the road of a partial and full serializer unless absolutely necessary, and I don't think it is in this case.